src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj

```diff
@@ -98,6 +98,10 @@
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
     <None Include="project.json" />
+    <Content Include="..\NuGet.PackageManagement.UI\Resources\PackageIconMonikers.imagemanifest">
+      <Link>PackageIconMonikers.imagemanifest</Link>
+      <IncludeInVSIX>true</IncludeInVSIX>
+    </Content>
```

Comment:
does it really need to ship as part of VSIX or can it included as an EmbeddedResource in the PackageManagement.UI dll?